### PR TITLE
References for my SPLC23 paper and some extra stuff

### PR DESCRIPTION
**Pull request by pmbittner**

### CODE REVIEW

File: SoftVarE-Publications/2021.bib [hunk 1]
```diff
@@ -0,0 +1,303 @@
+
+
+@inproceedings{T:SPLC21,
+  author = {Thomas Thüm},
+  title = {Keynote: Where Are My Constraints and What Do They Constrain?},
```
Comment (skrieter) [inline SoftVarE-Publications/2021.bib:5]: Title not protected

File: SoftVarE-Publications/2021.bib [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+  year = {2021},
```
Comment (skrieter) [inline SoftVarE-Publications/2021.bib:6]: should be without {}

File: SoftVarE-Publications/2021.bib [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+  month = {9},
```
Comment (skrieter) [inline SoftVarE-Publications/2021.bib:7]: should be SEP

File: SoftVarE-Publications/2021.bib [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+  booktitle = {Proc. Int'l Systems and Software Product Line Conf. (SPLC)},
+  address = {New York, NY, USA},
+  publisher = {ACM},
```
Comment (skrieter) [inline SoftVarE-Publications/2021.bib:10]: ACM

File: SoftVarE-Publications/2021.bib [hunk 5]
```diff
@@ -0,0 +1,303 @@
+
+
+@inproceedings{T:SPLC21,
+  author = {Thomas Thüm},
```
Comment (skrieter) [inline SoftVarE-Publications/2021.bib:4]: should be Th\"um, Thomas

File: SoftVarE-Publications/2021.bib [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+
+@inproceedings{RBTS:FMTea21,
+  author = {Tobias Runge and Tabea Bordis and Thomas Thüm and Ina Schaefer},
+  title = {Teaching Correctness-by-Construction and Post-Hoc Verification - The Online Experience},
+  year = {2021},
+  month = {11},
+  booktitle = {Proc. Int'l Workshop and Tutorial on Formal Methods Teaching (FMTea)},
+  address = {Berlin, Heidelberg, Germany},
+  publisher = {Springer},
+  pages = {101-116},
```
Comment (skrieter) [inline SoftVarE-Publications/2021.bib:27]: --

**Comment by pmbittner** [comment]
As discussed, the comments to `SoftVarE-Publications/2021.bib` do not apply because it is a generated file and is part of this PR for archiving purposes only. Sorry for not documenting that earlier.

**Comment by pmbittner** [comment]
The other changes are ok?

**Review by skrieter — APPROVED** [review]

**Pull request merged**